src/jit/compiler.cpp

```diff
@@ -2445,40 +2448,66 @@ void Compiler::compSetProcessor()
                             }
                         }
                     }
+#else  // ! FEATURE_CORECLR
+                    // If this is not FEATURE_CORECLR, the flags for SSE3, SSSE3, SSE41 and SSE42 are not set by the VM.
```

Comment:
Didn't we have the relevant CPUID checks for the `COMPlus_EnableSSE3_4` flag (that was introduced as part of the original `S.N.Vector` work several years ago)?

Reply:
Prior to the HWIntrinsics work, this was `jitFlags.IsSet(JitFlags::JIT_FLAG_USE_SSE3_4)`: https://github.com/dotnet/coreclr/blob/release/2.0.0/src/jit/compiler.cpp#L2457

Reply:
We got rid of the `USE_SSE3_4` flag in CoreCLR, in favor of the individual flags, but if the VM changes didn't flow back, it might still exist for full framework.

Reply:
Right - I think that's what confused me; I was thinking we had that but couldn't find it.